client/src/components/Common/HobbyItem/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './styles.css';
+
+const HobbyItem = ({ image, name, description }) => (
+  <div className="hobby-container row valign-wrapper">
```

Comment:
Note that `row` and `valign-wrapper` have a conflict in styles and are not working at the same time. I am fixing it in styles of the `hobby-container` class.